packages/newspringchurchapp/src/ui/WebBrowser/Browser.ios.js

```diff
@@ -2,16 +2,23 @@ import { Linking } from 'react-native';
 import SafariView from 'react-native-safari-view';
 
 const Browser = {
-  openURL: async (url, options) => {
-    try {
-      await SafariView.isAvailable();
-      SafariView.show({
-        url,
-        ...options,
-      });
-    } catch (e) {
+  openURL: async (url, { externalBrowser, ...options }) => {
```

Comment:
`externalBrowser` here needs a default (of false, probably) so that if nothing is passed in we won't get a "unhandled promise rejection" error.

Reply:
FWIW, that error is what I'm getting now on links that don't have the `externalBrowser` setting set. `Unhandled promise rejection TypeError: Cannot read property 'externalBrowser' of undefined`

Reply:
done